src/main/java/hng_java_boilerplate/sale/dto/ErrorResponse.java

```diff
@@ -0,0 +1,24 @@
+package hng_java_boilerplate.sale.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class ErrorResponse {
+    private Boolean status;
+    private int status_code;
+    private String error;
+    private String message;
+    private ErrorDetails details;
+
+    public ErrorResponse(Boolean status, int status_code, String error, String message){
```

Comment:
This isn't necessary, you already have the allArgsConstructor annotation

Reply:
Yes there is allArgsContructor, but i wanted another constructor without ErrorDetails

Reply:
Okay